web-client/src/views/CaseDetail/AddPrivatePractitionerModal.tsx

```diff
@@ -36,6 +36,7 @@ export const AddPrivatePractitionerModal = connect(
         className="counsel-modal"
         confirmLabel="Add to Case"
         confirmSequence={confirmSequence}
+        dataTestId="private-practitioner-modal"
```

Comment:
I think this should be `data-test-id`

Reply:
it's a prop on Modal which is why it's like that.